test/json.go

```diff
@@ -0,0 +1,24 @@
+package test
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+)
+
+// ReadJSON is a testing utility that reads data from JSON file into an
+// interface and fails with FailNow on error.
+//
+//   cfg := Config{}
+//   test.ReadJSON(t, "testdata/config.json", &cfg)
+func ReadJSON(t testing.TB, fname string, v interface{}) {
+	t.Helper()
+	f, err := os.Open(fname) // nolint:gosec
+	if err != nil {
+		t.Fatalf("cannot open %s: %v", fname, err)
```

Comment:
`os.Open` returns an `os.PathError` that is formatted (in this case) as ("open %s: %v", fname, err), so the error message is going to duplicate the op "open" and the filename. So you could format it as ("cannot %v", err), but that looks a little weird in the code, so maybe just `t.Fatal(err)` ?

Reply:
ok.